test/rpc-spec.js

```diff
@@ -1,17 +1,13 @@
 const assert = require('assert');
 const uuid = require('uuid');
 const arnavmq = require('../src/index')();
-const docker = require('./docker');
 const utils = require('../src/modules/utils');
 
 const fixtures = {
   queues: ['rpc-queue-0', 'rpc-queue-1', 'rpc-queue-2'],
 };
 
-describe('Producer/Consumer RPC messaging:', () => {
-  before(docker.rm);
-  before(() => docker.run().then(docker.start));
-
+describe('Producer/Consumer RPC messaging:', async () => {
```

Comment:
Is `async` needed here? I don't see `awat` used here...

Reply:
don't know why I put `async` here, removing it